study-group/README.md

```diff
@@ -0,0 +1,130 @@
+
+- Call To Action: button(s) displayed on the toast prompting user engagement.
+
+- Callbacks: functions triggered when the toast enters or exits certain states.
+
+- Events: JavaScript events triggered when the toast enters or exits certain states.
+
+- `icon`: pictograph displayed with the toast.
+
+- Progress Bar: element on toast displaying time remaining before timeout.
+
+### Built-In Support
+The Built-In Support establishes how a toast is defined and used, as well as what it's allowed to be used with.
+It consists of a collection of implementation details, patterns, and technologies that are supported by popular libraries.
+
+- Theme Support: offers built-in theming options, such as `success`, `info`, or `error`.
+
+- Global Custom Configuration: support for global config objects defining custom themes / patterns, which can be applied to individual toasts.
+
+- Configured / Created in JS: the toast object is set up and called to display using JavaScript.
+    - Upon display this creates a new DOM element or injects the settings into an existing HTML element.
+
+- Explicit Component Pattern: uses an element explicitly written into the HTML to display the toast when triggered.
+    - This element can be configured, or left as a dummy element to inject configuration into.
+
+- Permits Custom HTML: allows developers to write custom HTML templates, loaded in as children or passed as views.
+
+- Display in Container: allows the toast to display within a container, the default being `body`.
+
+- CSS Class Field: explicit field to pass in the CSS class or list of classes for styling the toast.
+
+- Explicit Animation Support: library offers an `animation` or `transition` field which abstracts away writing animations.
+
+- Swipe / Drag: the toast can be swiped or dragged to be dismissed, an important cross-platform consideration.
+
+- Intuitive Positioning: the library provides built in values to place your toast on the `top/middle/bottom` of the `left/center/right`.
+
+- User Interaction Trigger: tracks when user has moused over the toast, and allows the developer to change behavior when they do.
+
+### Policies
+The Policies of a toast library are what it chooses to allow, forbid, and prioritize. 
+It's the list of opinions about what a toast should be able to do.
+
+- Mentions Accessibility: library specifically mentions A11y, uses `aria`, or makes note of accessibility concerns (i.e. focus).
+    - **Note**: This was difficult to track, not tested comprehensively, and I could have made mistakes. Please create an issue if you notice an error.
+
+- Default Times Out: the toast dismisses by default, unless otherwise specified.
+
+- Allows Multiple: multiple toasts can be displayed at once, typically either by stacking or overlapping.
+
+- Allows Duplicates: the same toast can be called and displayed multiple times simultaneously.
+
+- Newest on Top: an option to toggle if the newest toast will show up above or below the previous toasts.
```

Comment:
This name is a bit confusing. Maybe it should be "Allows Newest on Bottom" or "Configurable Stacking" or similar? As phrased now it makes it sound like anything with an X here would _not_ show the newest on top.

Reply:
Definitely confusing, I rephrased it.

Reply:
changed language for 'Newest on Top' to 'Configurable Stacking'